src/wkt/src/message.rs

```diff
@@ -20,27 +20,52 @@ use crate::AnyError as Error;
 /// A trait that must be implemented by all messages.
 ///
 /// Messages sent to and received from Google Cloud services may be wrapped in
-/// [Any][crate::any::Any]. `Any` uses a `@type` field to encoding the type
+/// [Any][crate::any::Any]. `Any` uses a `@type` field to encode the type
 /// name and then validates extraction and insertion against this type.
 pub trait Message {
     /// The typename of this message.
     fn typename() -> &'static str;
 
+    /// Returns the serializer for this message type
     #[doc(hidden)]
```

Comment:
Heads up: we will also need to add a `#[allow(private_interfaces)]` annotation to this and every other `fn serializer()` after we make `MessageSerializer` private.

Reply:
Noted.